### ISSUE TITLE
Validation form is out of date for the demo assignment 3 

### ISSUE THREAD

**Issue by fralik** [Yorko/mlcourse.ai#371]
Questions 3.6 and 3.7 in the [validation form ](https://docs.google.com/forms/d/1wfWYYoqXTkZNOPy1wpewACXaj2MZjBdLOL58htGWYBA/edit) for demo assignment 3 are incorrect. The questions are valid for the previous version of the assignment that is accessible by commit 152a534428d59648ebce250fd876dea45ad00429.

**Comment by Yorko** [comment]
I don't see the point. Questions 3.6 and 3.7 are in total correspondence with the [assignment](https://mlcourse.ai/notebooks/blob/master/jupyter_english/assignments_demo/assignment03_decision_trees.ipynb?flush_cache=true).

**Comment by fralik** [comment]
The point is that anyone who tries to do demo assignments now will get their answers marked incorrect. Despite the answers being correct. Take me: I did the assignment, checked my answers and then boom! Answers I gave to 3.6 and 3.7 were incorrect. But the questions are relatively straightforward. I triple-checked my code. I was still getting the same values, which were incorrect according to the web form.

I then went on and checked out a previous version of the assignment. Run my code (exactly the same code). Hey! I got the correct answers.

Now to wrap up. Possible answers to question 3.6 from the web form are: `0.895; 0.856; 0.788; 0.845`.

Code with the answer to question 3.6 (corresponds to section 3.1 in the notebook)
```
tree = DecisionTreeClassifier(max_depth=3, random_state=17)
tree.fit(X_train, y_train)
tree_predictions = tree.predict(X_test)
accuracy_score(y_test, tree_predictions)
```

If one runs it in ed28f5f691fa3227aca62908bd316c8b70738376 (current master), the result is `0.790`.
If one runs it in 152a534428d59648ebce250fd876dea45ad00429 (previous version), the result is `0.845`.

Please note how the answer from the previous version matches web form expectations.

So I am saying that although this is a demo assignment, the validation form doesn't validate answers of the current version of the assignment.

**Comment by Yorko** [comment]
fixed that in 4130b0996e62c10f7091f2c117764f74b0809971

I've found the reason. Maybe it's a bug in sklearn implementation: accuracy depends on the order of columns. First categorical, then numeric features (1) or vice versa (2) - accuracy differs very much in (1) and (2). Now checked with docker, shall be all right.

Thanks!
